README.md

```diff
@@ -337,6 +337,90 @@ In the above case, `myapp` and `myapp2` containers will be instrumented, `myapp3
 
 > 🚨 **NOTE**: Go auto-instrumentation **does not** support multicontainer pods. When injecting Go auto-instrumentation the first pod should be the only pod you want instrumented.
 
+#### Multi-container pods with multiple instrumentations
+
+Works only when `operator.autoinstrumentation.multi-instrumentation` feature is `enabled`.
```

Comment:
Is the feature flag implemented because it can be a breaking change?

Reply:
If the feature flag breaks existing deployments we should make it clear here, or how to migrate

Reply:
Please see notes below under `annotations` - I believe all needed information related to configuration is provided.

Reply:
> If language instrumentation specific container names are not specified, instrumentation is performed on the first container available in the pod spec (only if single instrumentation injection is configured).

> only if single instrumentation injection is configured 

Do you mean then the feature flag is disabled? Or how is the single instrumentation configured?

Reply:
Ahh yes, this could be confusing. Single instrumentation is enabled by default and multi instrumentation flag is disabled by default. Also multi instrumentation means "multiple languages instrumentation for single or multiple containers" when single instrumentation stands for "single language instrumentation for single or multiple containers"